src/App.js

```diff
@@ -1,5 +1,6 @@
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import { Login } from "./Components/Auth/Login/login";
+import { SignUp } from "./Components/Auth/SignUp/signup";
 import { Navbar } from "./Components/Navbar/navbar";
```

Comment:
suggestion: export all components from one file (index.js) in components folder and import components from it